### ISSUE TITLE
sass-lint doesn't support simple js files as a valid configuration format

### ISSUE THREAD

**Issue by arcanis** [sasstools/sass-lint#731]
Two files formats are natively supported by Node.js: js and json. However, neither of these two formats are available to be used as configuration. Could it be possible to add them? You would just have to check that the extension is a key in the `require.extensions` dict, and use `require()` in such a case.

**Comment by DanPurdy** [comment]
This is true, we do plan to add support soon and there's already a PR i believe that goes part of the way.

The original reason for the format and the config naming was to mimic the scss-lint way so as to keep it familiar for people who adopted this project early. It's just never been updated for others.
